test/bigtest/network/config.js

```diff
@@ -88,7 +90,31 @@ export default function config() {
           module: MODULE_ORDERS,
           configName: CONFIG_ORDER_NUMBER,
           enabled: true,
-          value: '{"canUserEditOrderNumber":false,"selectedPrefixes":["PP"],"prefixes":["PP1","PP2","PP3","PP"],"selectedSuffixes":["SS"],"suffixes":["SS1","SS2","SS"]}',
+          value: '{"canUserEditOrderNumber":false}',
+        }],
+      };
+    }
+
+    if (queryParams && queryParams.query === `(module=${MODULE_ORDERS} and configName=${CONFIG_PREFIXES})`) {
```

Comment:
I think it makes sense to use factories here and create required settings in corresponding tests